docs/docs/handbook/student_management.md

```diff
@@ -60,6 +61,36 @@ To create new students in your tutorial click the "+ New" button on the upper ri
 
 1.  **Abort button**: Closes the form without saving the information.
 
+## Import Students
+
+To import students from a CSV file click on the "<IconInText icon={GenerateIcon} /> Generate" button in the upper right. A step-by-step wizard will guide you through the import process.
+
+1. **Import CSV content**: Import the CSV content by either copy & paste the content in the input field or by uploading the file. You can upload the file by clicking the "Upload CSV-file" button and the content will be pasted into the input field after uploading. If the separator is not correctly detected on the next page you can change the seperator on _this_ page using the "Separator" input in the upper right.
+
+   :::tip Drag & Drop
+   You can also drag & drop the file on the button
+   :::
+
+1. **Map Columns**: The TMS tries to find all needed columns automatically. If the columns are not properly detected you can change them through the dropdown menus on this page. The following columns are supported (_required_ ones must be provided):
+
+   - _Firstname (**required**)_: The firstname of the student.
+
+   - _Lastname (**required**)_: The lastname of the student.
+
+   - _Status_: The status of the student, one of `ACTIVE`, `INACTIVE`, and `NO_SCHEIN_REQUIRED`. Matches the provided value case-insensitive. If no value is provided, the status is set to `ACTIVE`.
+
+   - _Team_: An identifier of the team the student is put into. If a number is provided, and a team with that number already exists, the student is put into that team. For all other identifiers, new teams are created. If no value is provided, the student is put into no team.
+
+   - _E-Mail Address_: A valid e-mail address of the student.
```

Comment:
Should "address" be written in lower case?

Reply:
I copied it from [here](https://github.com/Dudrie/Tutor-Management-System/blob/main/docs/docs/handbook/user_management.md?plain=1#L77).
If you want, I can change both.

Reply:
That was an oversight on my end at that time. If you would change both it would be good.

Reply:
changed with f070b1d69d9b43a215c603b5463d9dec7973bf0f